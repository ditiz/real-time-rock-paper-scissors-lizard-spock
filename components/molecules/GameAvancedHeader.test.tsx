import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import GameAdvancedHeaderMolecule from "./GameAvancedHeader"

it("renders GameAdvancedHeaderMolecule unchanged", () => {
  const { container } = render(<GameAdvancedHeaderMolecule />)
  expect(container).toMatchSnapshot()
})

it("should contain title", () => {
  render(<GameAdvancedHeaderMolecule />)
  expect(screen.getByText(/Rock Paper Scissors Lizard Spock/))
})
