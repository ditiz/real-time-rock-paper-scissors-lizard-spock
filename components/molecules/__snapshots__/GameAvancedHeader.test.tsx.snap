// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GameAdvancedHeaderMolecule unchanged 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      Rock Paper Scissors Lizard Spock
    </h1>
  </header>
</div>
`;
